accounts/pkg/storage/cs3.go

```diff
@@ -206,12 +206,11 @@ func (r CS3Repo) DeleteGroup(ctx context.Context, id string) (err error) {
 
 func (r CS3Repo) authenticate(ctx context.Context) (token string, err error) {
 	u := &user.User{
-		Id:     &user.UserId{},
+		Id: &user.UserId{
+			OpaqueId: r.cfg.ServiceUser.UUID,
```

Comment:
@kulmann we can always set tha uuid ... if it is "" that is the same as this if magic ;-)

Reply:
It is not empty by default. It has a default UUID value in the flatset, so that it `just works` as soon as the username is provided. You're right, but it needs more change than this. ;-) Clear the default value for the UUID in the flagset and replace the checks on the username flag with checks on the uuid flag.

Reply:
yes it has a uuid, but before this change it would only set it if the username was also set ... but it is empty by default: https://github.com/owncloud/ocis/pull/657/files#diff-69d2327c70e84f0e2b037204de3ef8e3L212-L214